css/style.css

```diff
@@ -38,13 +38,25 @@
   padding-left: 2rem
 }
 
+  /* cards */
+  .all-shows-card-name {
+    display: -webkit-box;
```

Comment:
`display: -webkit-box` is non-standard from a time before `display: flex` existed. You should use the more modern and standardised `display: flex` instead.